scheduler/src/org/pentaho/platform/scheduler2/email/Emailer.java

```diff
@@ -166,6 +167,7 @@ public boolean setup() {
       props.put( "mail.smtp.ssl", ObjectUtils.toString( service.getEmailConfig().isUseSsl() ) );
       props.put( "mail.smtp.quitwait", ObjectUtils.toString( service.getEmailConfig().isSmtpQuitWait() ) );
       props.put( "mail.from.default", service.getEmailConfig().getDefaultFrom() );
+      props.put( "mail.debug", true );
```

Comment:
This should either not be here or it should be a configurable option that is disabled by default.